src/Symfony/Component/HttpFoundation/FileBag.php

```diff
@@ -84,7 +84,7 @@ protected function convertFileInformation($file)
                 if (UPLOAD_ERR_NO_FILE == $file['error']) {
                     $file = null;
                 } else {
-                    $file = new UploadedFile($file['tmp_name'], $file['name'], $file['type'], $file['size'], $file['error']);
+                    $file = new UploadedFile($file['tmp_name'], $file['name'], $file['type'], null, $file['error']);
```

Comment:
we should deprecate passing the arg in fact, isn't it?

Reply:
This is what we are doing. But I can't remove it until 5.0, or can I?

Reply:
I missed that on the merged PR!
Yet we can, and we must in fact.
Right now, we are forced to pass a "null", which is not the signature we want on v5

Reply:
(but we must provide a BC layer of course)

Reply:
let's do it then ;)

Reply:
done